### PR TITLE
volunteer personal hours page - filter jobs by contact

### PR DESCRIPTION
**Pull request by daizylogik**
Hi David,
This is the last enhancement to the package, allowing the jobs dropdown on the personal site report hours to be filtered by contact.
Thanks. Vladimir

### CODE REVIEW

**Comment by daizylogik** [comment]
@davidhabib Hi David,
Let me know if you had a chance to take a look at the last pull request, and if there is anything you would like me change. Thanks. Vladimir

File: src/classes/VOL_CTRL_VolunteersReportHours.cls [hunk 1]
```diff
@@ -123,16 +123,48 @@ global virtual with sharing class VOL_CTRL_VolunteersReportHours {
...
+				} else {
+					volunteerJobs = [select Name, Id
+				        			from Volunteer_Job__c
+				        		 	where Campaign__r.IsActive = true and Inactive__c = false 
+				        		 	order by name limit 999];
+				}
+				
+				for (Volunteer_Job__c vj : volunteerJobs) {
+					if(filterByContact && contactIdPersonalSite != null) {
+						for (Volunteer_Hours__c volHour : vj.Volunteer_Hours__r) {
```
Comment (davidhabib) [inline src/classes/VOL_CTRL_VolunteersReportHours.cls]: won't this end up adding the job to the list once per hours?  so if the person had multiple hours records under the job (ie, maybe they worked various shifts), then the job would get duplicated in the Job dropdown?

Comment (sgjudd) [inline src/classes/VOL_CTRL_VolunteersReportHours.cls]: David, It only includes the job once. The sub-select to get hours for the contact has Limit 1.

Comment (davidhabib) [inline src/classes/VOL_CTRL_VolunteersReportHours.cls]: ah, I missed the limit 1.  darn line wrapping in gitHub put it on the next line far left!

**Pull request merged**